tests/test_game_logic.py

```diff
@@ -1,63 +1,59 @@
 import pytest
-from ..game_logic import check_bingo, manage_turns
-from ..bingo_card_generation import generate_bingo_card, display_bingo_card
-
-
+import game_logic
```

Comment:
`pytest` runs tests from the root directory, which is available in `sys.path`, so we do not need to use relative imports (`..game_logic`) here.